### ISSUE TITLE
Rating - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570024)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7800]
### Violation: 
 Ensure parts of graphical objects essential for understanding content have sufficient contrast 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/1a3b5f98-5e6f-48a5-8694-4ad49c61ad29)

### WCAG Reference: 
- [1.4.11 Non-test Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) 

### Severity: 
 5 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: Rating - Ensure parts of graphical objects essential for understanding content have sufficient contrast - (2036570024) 
- Module: Rating 
**Issue**
There are graphical objects with a color contrast ratio below 3.00:1 on hover.

Foreground: #7FBADC
Background: #7FBADC
Contrast ratio: 2:1

Examples include:
- Rating start images on hover 

**User Impact**
Users with low vision will have difficulty identifying this content.

 
**Suggestion**
Ensure parts of graphical objects essential for understanding content have sufficient contrast. The required minimum contrast ratio is 3.00:1.

Common examples of qualifying objects include lines in a chart, meaningful icons, and annotations within an image.

Graphics that require particular presentation to preserve their meaning are exempt from this requirement.

To evaluate color contrast, see the Accessible Color Picker extension: https://www.accessibility.dev/

 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This is an invalid test, sufficient contrast is met with the following [color scheme](https://webaim.org/resources/contrastchecker/?fcolor=007AC2&bcolor=FFFFFF) with a contrast ratio of 4.6:1 and focus state 
![image](https://github.com/Esri/calcite-design-system/assets/5023024/75a4b835-eea0-4320-9210-9c70b4a22f37)
